nss-day-cohort-46/rare-server-tragic-trolls
# Nc editing a post

# Description

Made a post editable. It will update the posts and postTags tables. The delete post now deletes related rows in postTags table.

## Type of change


# Testing Instructions

Ensure you have some data in your posts table.

Do a PUT method in Postman with this info:
```
http://localhost:8088/posts/YOUR-POST-ID-THAT-YOU-WANT-TO-EDIT
```
```
{
        "title": "Updated tag list and post",
        "imageUrl": "",
        "tagIds": [YOUR LIST OF EXISTING TAG IDS],
        "userId": 2,
        "categoryId": 1,
        "content": "Post Text",
        "publicationDate": "2026-04-02T23:57:07.374Z",
        "approved": true
}
```
Do a GET method in postman using:
```
http://localhost:8088/posts/YOUR-POST-ID-THAT-YOU-JUST-EDITED
```
See that it contains the updated information.

# Checklist:
